Day-I-Social-Card/social-card/src/components/HeaderComponents/HeaderTitle.js

```diff
@@ -1,2 +1,16 @@
 import React from 'react';
 import './Header.css';
+
+
+const HeaderTitle = () => {
+  return (
+    <div className='header-title'>
+      <p className='name'>
+        Lambda School </p>
+        <p className='handle'>@LambdaSchool </p>
```

Comment:
this is a great start to the social card. Now let's focus on the main content and the footer and you're good to go.